### PR TITLE
Decreased real_time_update_rate from 400 to 160Hz

### PR DESCRIPTION
**Pull request by damianacatanoso**
See: https://github.com/nasa/ow_simulator/pull/37

### CODE REVIEW

File: worlds/atacama_y1a.world [hunk 1]
```diff
@@ -17,11 +17,11 @@
...
       <ode>
         <solver>
-          <type>quick</type>
+          <type>world</type>
         </solver>
       </ode>
-      <max_step_size>0.0025</max_step_size>
-      <real_time_update_rate>400</real_time_update_rate>
+      <max_step_size>0.00625</max_step_size>
+      <real_time_update_rate>160</real_time_update_rate>
```
Comment (mogumbo) [inline worlds/atacama_y1a.world]: I'm concerned that this is too close to the edge of stability. Since we can't test every possible arm movement, there might be a corner case that gives us instability that we just haven't discovered yet. Should we leave some breathing room by running at 180Hz or 200Hz instead?

Comment (damianacatanoso) [reply worlds/atacama_y1a.world]: I think you are right, this is the lowest I could get, and it might be unsafe for future developments. I'd go with 200Hz, such that the inverse is a finite number (1/200=0.005) and we can get a real time factor of exactly 1.

Comment (mogumbo) [reply worlds/atacama_y1a.world]: 200 Hz sounds safe to me. It will still be double the performance that we had before.

Comment (damianacatanoso) [reply worlds/atacama_y1a.world]: Done!

**Review by mogumbo — APPROVED** [review]

**Review by kmdalal — APPROVED** [review]
I don't understand why this PR is separate from #37, as they both are for the same branch/Jira and repository.   It did however show me where rates are encoded, which I didn't know before.  I'm approving this without any further testing.

**Review by Samahu — APPROVED** [review]
For now this seem fine

**Comment by damianacatanoso** [comment]
> I don't understand why this PR is separate from #37, as they both are for the same branch/Jira and repository. It did however show me where rates are encoded, which I didn't know before. I'm approving this without any further testing.

A single PR cannot contain multiple branches. In this case I have two branches: one for ow_simulator and one for ow_europa. If I want you to review both, I have to create two PRs. In Fisheye and Crucible we could include multiple branches in the same code review. Here we can't.
